### PR TITLE
ch01: fix nit in ESLint --fix result example

### PR DESCRIPTION
**Pull request by vsemozhetbyt**
This error is mentioned in the screenshot above and is fixed in my local `--fix` run.

### CODE REVIEW

File: ch01.asciidoc [hunk 1]
```diff
@@ -311,7 +311,7 @@ When we run `lint-fix` we'll only get a pair of errors: `hello` is never used an
 ----
 var goodbye = 'Goodbye!'
 
-function hello() {
+function hello () {
```
Comment (ljharb) [inline ch01.asciidoc:314]: Is this intentional? It’s very strange to have a space between the function name and the argument list.

Comment (vsemozhetbyt) [reply ch01.asciidoc:314]: See https://github.com/mjavascript/practical-modern-javascript/blob/master/ch01.asciidoc#fig0102

> Missing space before function parentheses

Maybe Standard activates https://eslint.org/docs/rules/space-before-function-paren ?

Comment (ljharb) [reply ch01.asciidoc:314]: lol, if so, it’s just another item to add to the list of how nonstandard “standard” is ¯\\\_(ツ)_/¯

**Comment by bevacqua** [comment]
Thanks!

**Pull request merged**